supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesBackendLifecycleActorFactory.scala

```diff
@@ -4,13 +4,21 @@ import java.nio.file.Path
 
 import akka.actor.Props
 import com.typesafe.config.Config
-import cromwell.backend.impl.jes.io._
-import cromwell.backend.{BackendConfigurationDescriptor, BackendJobDescriptor, BackendJobDescriptorKey, BackendLifecycleActorFactory, BackendWorkflowDescriptor}
+import cromwell.backend._
 import cromwell.core.{ExecutionStore, OutputStore}
 import wdl4s.Call
 import wdl4s.expression.WdlStandardLibraryFunctions
 
+import scala.language.postfixOps
+
+object JesBackendLifecycleActorFactory {
+  implicit class Jessify(val genericInitializationData: Option[BackendInitializationData]) {
+    def toJes = genericInitializationData collectFirst { case d: JesBackendInitializationData => d } get
```

Comment:
Is danger your middle name, is that get actually safe, or did you just throw your hands up in the air like you, well, you know